cs/configuring.texy

```diff
@@ -345,3 +344,17 @@ Například takto může vypadat definice služby pojmenované `database`, což
 services:
 	database: PDO('mysql:host=127.0.0.1;dbname=test', root, password)
 \--
+
+
+Rozšíření
+=========
+
+Pomocí rozšíření můžeme rozdělit naši aplikaci do malých nezavislých častí s vlastní konfigurací. Nette obsahuje řadu předdefinovaných rozšíření, o kterých se dočtete více v kapitole [DI: Základní rozšíření pro kontejner |di-defaultextensions].
```

Comment:
Ono to tak demonstruje i samotny Nette, ze mas application, database, http v samostatnych balickach. Pres CompilerExtension se vzdy nastavi sluzby, co tam maji byt a pripadne se resi nejaka provazanost. 

Nebo rekni, jak by jsi to chtel ty. Neprijde mi, ze by ten text nejak vyslovene lhal, ne?